### DISCUSSION TITLE
npm not installing

### DISCUSSION THREAD

**Discussion by Abhishek0732** [HimanshuNarware/Devlabs#229] [General]
Why the npm install command is not installing all the dependencies

PS C:\Users\asus\Desktop\GSSOC\Devlabs> npm install

up to date, audited 2 packages in 846ms

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities

showing this type

**Comment by HimanshuNarware** [comment]
@Abhishek0732 show me screenshot

**Comment by Abhishek0732** [comment]
![Screenshot (160)](https://github.com/HimanshuNarware/Devlabs/assets/93417069/71809de7-3633-4e69-baf9-e5dc68b1e480)

**Comment by Abhishek0732** [comment]
please help me to get out this error

**Reply by HimanshuNarware** [reply]
why you installing npm fund ,  just start development by writing cmd npm start

**Comment by HimanshuNarware** [comment]
No need to type npm fund ok

**Comment by Abhishek0732** [comment]
![Screenshot (162)](https://github.com/HimanshuNarware/Devlabs/assets/93417069/15f4e835-1135-48e0-814f-d3017cc659c8)

Then also getting the error

**Comment by HimanshuNarware** [comment]
Update you Nodejs

**Comment by HimanshuNarware** [comment]
do one thing send me the screenshot of nodejs version
 and  type ls command  in the folder where you have clone the devlabs and send screenshot

**Reply by Abhishek0732** [reply]
![Screenshot (163)](https://github.com/HimanshuNarware/Devlabs/assets/93417069/8f0d4a1b-fa12-4002-b6be-4ffc345df4e5)

**Comment by Abhishek0732** [comment]
![Screenshot (164)](https://github.com/HimanshuNarware/Devlabs/assets/93417069/2a72ca2e-e42a-4094-b17d-37599fecc402)

version

**Comment by HimanshuNarware** [comment]
@Abhishek0732  if you want to contribute to  the frontend of the Devlabs , install npm dependencies in frontend folder of devlabs, and if you want to work on backend then you  have to install depencies on backend

**Comment by Abhishek0732** [comment]
@HimanshuNarware please check i have created the pull request

**Comment by kushalkumar1362** [comment]
Install npm in correct folder cd frontend

**Reply by Abhishek0732** [reply]
done

**Comment by vanisharma2003** [comment]
yes write npm install in cd frontend as it has package.json file. By writing npm start here you can also view project in your desktop.
Happy learning.
